patroni/postgresql.py

```diff
@@ -717,7 +717,9 @@ def start(self, timeout=None, block_callbacks=False):
         # In order to make everything portable we can't use fork&exec approach here, so  we will call
         # ourselves and pass list of arguments which must be used to start postgres.
         proc = call_self(['pg_ctl_start', self._pgcommand('postgres'), '-D', self._data_dir] + options, close_fds=True,
-                         preexec_fn=os.setsid, stdout=subprocess.PIPE, env={'PATH': os.environ.get('PATH')})
+                         preexec_fn=os.setsid, stdout=subprocess.PIPE,
+                         env={'PATH': os.environ.get('PATH'), 'LC_ALL': os.environ.get('LC_ALL'),
```

Comment:
Well, if it's not defined, then .get will get you None, nothing wrong here. I think your suggestion makes sense, on the second look I'm wondering why do we clear the environment altogether?